tests/e2e/hcp_cluster_test.go

```diff
@@ -182,4 +182,35 @@ var _ = Describe("HCP cluster testing",
 					Should(ContainSubstring(
 						"ERR: Listing identity providers is not supported for clusters with external authentication configured"))
 			})
+
+		It("can edit ROSA HCP cluster with additional allowed principals - [id:74456]",
+			labels.High, labels.Runtime.Day2,
+			func() {
+				By("Check the help message of 'rosa edit cluster -h'")
+				helpOutput, err := clusterService.EditCluster("", "-h")
+				Expect(err).To(BeNil())
+				Expect(helpOutput.String()).To(ContainSubstring("--additional-allowed-principals"))
+
+				By("Check if cluster profile is enabled with additional allowed priincipals")
+				if clusterConfig.AdditionalPrincipals == "" {
+					SkipTestOnFeature("additional allowed principals")
+				}
+
+				output, err := clusterService.DescribeClusterAndReflect(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(output.AdditionalPrincipals).To(Equal(clusterConfig.AdditionalPrincipals))
+
+				By("Edit the cluster with additional allowed principals")
+				out, err := clusterService.EditCluster(clusterID,
+					"--additional-allowed-principals",
+					clusterConfig.AdditionalPrincipals)
```

Comment:
Shouldn't we add more principal ? Like create new account roles and use the Installer Role ?

Reply:
Since this is just exercising the rosa cli, I don't think it matters which role we use (or reuse). Using the basic installer role will be fine to verify that the value is set on the cluster itself.

Reply:
Or are you referring to adding more than 1 role?

Reply:
Thnaks @radtriste @jfrazierRH I changed the code likewise PTAL again, Thanks

Reply:
@jfrazierRH Yes I was referring to add more roles than existing, to make sure the CLI pass them accordingly